### PR TITLE
osdocs392 bootstrap gather draft

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-392

built draft here: http://file.rdu.redhat.com/kalexand/050819/osdocs392/installing/installing-gather-logs.html

I added the ssh key and agent steps to all the install topics as an optional process for production clusters, eg: http://file.rdu.redhat.com/kalexand/050819/osdocs392/installing/installing_aws/installing-aws-default.html#ssh-agent-using-installing-aws-default

@jstuever, will you PTAL?

### CODE REVIEW

**Comment by jstuever** [comment]
Just a few tweaks to the "Gathering installation logs" page

- Prerequisite: the bootstrap node must be running and accessible over SSH. IPI will automatically destroy the bootstrap node after "bootstrap-complete".
- Procedure: After the first step and before downloading the tarball, the installer-gather.sh script must actually be executed on the boostrap node. The syntax is the second line in the output of the first step (ssh -A core@...). The first step only outputs the commands to be run by the user and is called automatically in IPI when bootstrap-complete fails. In the future, we intend to extend "gather bootstrap" to automagically call the installer-gather.sh script as well as download the tarball. Until then, they remain as additional steps for the user.

**Comment by kalexand-rh** [comment]
Thanks @jstuever! 
@shlao, when you've approved the engineering JIRA, will you PTAL at the docs?

File: modules/installation-bootstrap-gather.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-bootstrap-gather-{context}"]
+= Gathering logs from a failed installation
+
+If you gave an SSH key to your installer, you can gather data about your failed
+installation.
+
+[NOTE]
```
Comment (kalexand-rh) [inline modules/installation-bootstrap-gather.adoc]: @sferich888, based on the discussion from https://github.com/openshift/openshift-docs/pull/14811, do you think that this note is appropriate?

**Comment by kalexand-rh** [comment]
@shlao, will you PTAL? We're trying to get all of the content for 4.1 GA merged this week.

**Comment by shlao** [comment]
@kalexand-rh  LGTM

**Comment by kalexand-rh** [comment]
Thank you! Merging.

**Pull request merged**